frontend/src/components/homepage/StoryCard.tsx

```diff
@@ -134,16 +144,16 @@ const StoryCard = ({
           <p className="story-card-title">{title}</p>
           <div className="story-card-tags">
             {/* TODO: make tags that reflect actual state */}
-            <Tag displayText={`level ${level}`} />
-            <Tag displayText="language needed" />
+            <Tag displayText={`Level ${level}`} />
```

Comment:
did design have any preference for capitalization?

Reply:
Good question. Will double-check with them.